tensorboard/webapp/widgets/data_table/data_table_component.ts

```diff
@@ -177,4 +189,15 @@ export class DataTableComponent {
         return '';
     }
   }
+
+  headerClicked(header: ColumnHeaders) {
+    let sortingOrder = SortingOrder.ASCENDING;
+    if (
+      this.sortingInfo.header === header &&
+      this.sortingInfo.order === SortingOrder.ASCENDING
+    ) {
+      sortingOrder = SortingOrder.DESCENDING;
+    }
+    this.sortDataBy.emit({header, order: sortingOrder});
+  }
```

Comment:
This is a nitpick, but storing and mutating the variable seems unnecessary.

```suggestion
  headerClicked(header: ColumnHeaders) {
    if (
      this.sortingInfo.header === header &&
      this.sortingInfo.order === SortingOrder.ASCENDING
    ) {
      this.sortDataBy.emit({header, order: SortingOrder.DESCENDING});
      return;
    }
    this.sortDataBy.emit({header, order: SortingOrder.ASCENDING});
  }
```